lib/index.js

```diff
@@ -113,6 +116,14 @@ exports.publish = function publish(basePath, config, callback) {
     return;
   }
 
+  if (options.cname) {
+    fs.writeFileSync(path.join(basePath, 'CNAME'), options.cname);
+  }
```

Comment:
I think this will result in the `CNAME` file being written to the `dest` directory (which may be a subdirectory not at the root of the repo).  I think it should be written to the root instead.